### PR TITLE
upper/db hello world tutorial

### LINKED ISSUE #8048: Review upper/db tutorial and add it to "Hello World" examples
Tutorial draft [here](https://docs.google.com/document/d/12EpzNghXaHbJCIHLdpmA_dPW2B3H-eudgxspzxGJyGE/edit#)

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes #8048.

@vy I'm not sure what language we want to use to refer to upper/db. I categorize it in the tooling tables as an ORM, but call it a "data access tool" elsewhere, because that's how they refer to it on their site. Thoughts?

### CODE REVIEW

**Comment by vy-ton** [comment]
@ericharmeling I would like to rename the table to `Data access frameworks - Ex. object-relational mappers (ORMs)`

File: v20.2/build-a-go-app-with-cockroachdb-upperdb.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+Create a certificate and key for the `maxroach` user by running the following command:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ cockroach cert create-client maxroach --certs-dir=certs --ca-key=my-safe-directory/ca.key
+~~~
+
+The code samples will run with `maxroach` as the user.
+
+## Step 3. Create the `accounts` table
```
Comment (vy-ton) [inline v20.2/build-a-go-app-with-cockroachdb-upperdb.md:38]: @xiam is it possible for upper/db to create the `accounts` table in the code?

Comment (xiam) [reply v20.2/build-a-go-app-with-cockroachdb-upperdb.md:38]: Sure, we can integrate this step into the code

Comment (xiam) [reply v20.2/build-a-go-app-with-cockroachdb-upperdb.md:38]: @vy-ton I updated that bit in https://github.com/cockroachlabs/hello-world-go-upperdb/pull/2

**Comment by xiam** [comment]
@vy-ton, @ericharmeling do you think we can use Data Access Layer (or DAL)? Instead of tool. I think that is more accurate. Thanks!

**Comment by ericharmeling** [comment]
> @vy-ton, @ericharmeling do you think we can use "Data Access Layer" (or DAL)? Instead of "Data Access Tool". I think that is more accurate. Thanks!

Sure!

**Comment by ericharmeling** [comment]
@xiam @vy 

If everything looks good to y'all, passing off to writer review!

**Review by lnhsingh — COMMENTED** [review]
LGTM, nice job!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8410#-:-MHlKgCdBPCCiDpCMEYs:b-awek4m)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @lnhsingh and @vy-ton)

**Comment by ericharmeling** [comment]
TFTR @lnhsingh !

**Comment by xiam** [comment]
So excited! Thank you all

**Comment by ericharmeling** [comment]
Thanks for iterating on this @xiam !

**Pull request merged**